domains/imsoumyaneel.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "A-Little-Hat",
+        "email": "soumyasarkar309@gmail.com"
+    },
+    "record": {
+        "CNAME": "test-3e8b2.web.app/"
```

Comment:
nevermind i just rechecked and your CNAME record is invalid, can not contain `/`